### ISSUE TITLE
Friction log when using helm to install Istio

### ISSUE THREAD

**Issue by andraxylia** [istio/istio.io#1299]
Followed this:

https://preliminary.istio.io/docs/setup/kubernetes/helm-install.html

- missing instructions on how to install helm

- Not sure what this is, followed the link but not clear what is the action

	•	"If you want to manage Istio releases with Tiller, the capability to install service accounts is required."

- very misleading, it indicates sidecar injector must be installed before anything
Prerequisites: sidecar injection 
This should be moved after the main istio.install, but I get an error when I run it:
$helm install --namespace=istio-system --set sidecar-injector.enabled=true install/kubernetes/helm/istio
Error: release loopy-stoat failed: configmaps "istio-mixer-custom-resources" already exists

- missing instruction to download the latest release before running the helm template command

- missing what is next the link to bookinfo demo with alpha3
What’s next
See the sample Bookinfo application.

**Comment by sdake** [comment]
@andraxylia perhaps the docs have been updated, but the docs state:

helm install install/kubernetes/helm/istio --name istio --namespace istio-system --set sidecar-injector.enabled=true

the name of the deployment is required.  This works for with the 5-18 build.

**Comment by sdake** [comment]
RE: ```If you want to manage Istio releases with Tiller, the capability to install service accounts is required.``` - the action is to enable the capability to install service accounts for this step:
```kubectl create -f install/kubernetes/helm/helm-service-account.yaml
```

**Comment by sdake** [comment]
Note working from these docs: https://preliminary.istio.io/docs/setup/kubernetes/helm-install.html

**Comment by sdake** [comment]
@andraxylia - the docs have undergone significant revision by the community and the implementation is significantly different from when this issue was filed.  Do these problems still persist?  If so, could clarify what problems exist with the documentation?

Cheers
-steve

**Comment by sdake** [comment]
@andraxylia closing the helm friction log as I believe all items have been addressed or are recorded in other larger works (such as the SRE guide).  If you believe this was closed in error, feel free to reopen this issue or open a new issue with a new set of requests.

Cheers
-steve
